src/drt/src/io/io.cpp

```diff
@@ -1810,9 +1771,8 @@ void io::Parser::setMacros(odb::dbDatabase* db)
             }
           }
         }
-        auto blkIn = make_unique<frBlockage>();
+        auto blkIn = make_unique<frBlockage>(obs);
```

Comment:
> Does it make more sense to store the `dbObstruction` than the bbox of the obstruction?
I am trying to be consistent with the object used in this code block which is of type bbox